client/src/components/adCarrier/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
```

Comment:
Be very careful when you're switching branches not to bring any changes/files from another branch with you! Also, check and make sure what you commit is what should be committed before you do. 

To delete this file from this branch just make sure you're on the shuffle button branch and then delete this file from the codebase, then push that change. It won't affect the file on the other branch so no worries!